frontend/src/components/core/Block/Block.tsx

```diff
@@ -160,50 +136,68 @@ class Block extends PureComponent<Props> {
     )
   }
 
+  private static getClassNames(isStale: boolean, isEmpty: boolean): string {
+    const classNames = ["element-container"]
```

Comment:
We have a module classNames that you could use https://www.npmjs.com/package/classnames, but it is ok if you want to leave it as this

Reply:
Ah... ok,... now I understand what you meant in your first comment!